src/oscar/apps/shipping/abstract_models.py

```diff
@@ -100,8 +100,10 @@ class AbstractWeightBased(AbstractBase):
     # The attribute code to use to look up the weight of a product
     weight_attribute = 'weight'
 
-    code = 'weight-based-shipping'
-    name = _('Weight-based shipping')
+    # Had to be removed to get test to run
+    # Are we just setting defaults or titles here?
+    # code = 'weight-based-shipping'
+    # name = _('Weight-based shipping')
```

Comment:
I could do with some help on this part. I couldn't figure out how to get the tests to pass without commenting this out completely.